tests/console/consoletest_setup.pm

```diff
@@ -100,6 +100,11 @@ sub run() {
         assert_script_run("grub2-mkconfig -o /boot/grub2/grub.cfg");
     }
 
+    # https://trello.com/c/Y57ujlSn/663-3-plan-b-for-sle-12-sp2-p1-988157-hps-bug-sles12-sp2-install-gets-stuck-initializing-on-a-8-blade-hpe-bl920-gen-9-system
+    if (check_var('NETWORK_INIT_PARAM', 'ifcfg=eth0=dhcp')) {
+        assert_script_run "less /var/log/YaST2/y2log*|grep \"Automatic DHCP configuration not started - an interface is already configured\"";
```

Comment:
why `less .* | grep` and not just `grep "..." /var/log/YaST2/y2log*`? less piped to grep is pretty unusual and might do something unexpected e.g. when content covers more than one page, isn't it?